### DISCUSSION TITLE
Is view of mailbox  module how it should be?

### DISCUSSION THREAD

**Discussion by jojo12** [slabbi/uddeIM#96] [Q&A]
When I remember right, then the view of the module changed with the template. But now it seems to be always the same.
Is this correct or an error?

**Comment by joomod** [comment]
yes and no.
the module is still loading the uddemodule.css depending on the template.
what I have done was to stack the icons from the default template, as some of
the new templates have mixed the icons (transparent and background icons)
but you may change this in the script:
`//$iconpath = $uddpathtosite.'/components/com_uddeim/templates/'.$uddconfig->templatedir.'/images';
$iconpath = $uddpathtosite.'/components/com_uddeim/templates/default/images';`
just **outcommand the first** line (with $uddconfig->templatedir) and **command the second**

**Comment by joomod** [comment]
I edit this module and add the option to select icons (default template) or **fontawesome**

![uddmailbox](https://github.com/slabbi/uddeIM/assets/149375504/8ef48d36-ddf6-4978-9831-7f5bf31ac50b)

**Comment by inLIMITit** [comment]
Wow, Thank you!
